docs/errors/E0450.md

```diff
@@ -0,0 +1,20 @@
+# E0450: misleading use of comma operator in indexing operation
+
+Using comma operator within an indexing operation is misleading, as it
+can be confused with a new array definition, or with access to multiple elements
+of the container.
+
+For example:
+
+```javascript
+let a = [1, 3, 5, 7, 9, 11];
+let b = a[0, 1, 2];
+```
+Here, the value of `b` will be equal to `a[1] = 5`.
+If the intention is to create a new array `b` with the first three elements of array `a` the correct syntax would be:
+```javascript
+let a = [1, 2, 3, 4];
+let b = a[a[0], a[1], a[2]];
+```
+
+Introduced in quick-lint-js version 2.11.0.
```

Comment:
Must fix: This is wrong. Version 2.11.0 has already been released.

I'd just delete this paragraph for now:
```suggestion
```